python/cugraph/structure/symmetrize.py

```diff
@@ -16,7 +16,7 @@
 import dask_cudf
 
 
-def symmetrize_df(df, src_name, dst_name):
+def symmetrize_df(df, src_name, dst_name, multi=False, symmetrize=True):
```

Comment:
We should discuss rather we symmetrize on dataframe or if a graph class should be passed in

Reply:
To be added in a separate PR